mod/wet4/views/default/page/elements/head.php

```diff
@@ -259,16 +259,41 @@
 }
 
 // Meta tags for the page
+
+if (!$my_page_entity instanceof ElggEntity) {
+	$segments = substr($_SERVER['REQUEST_URI'], 0, strpos($_SERVER['REQUEST_URI'], '?'));
+	$segments = explode('/', $segments);
+	$some_post = get_entity($segments[sizeof($segments) - 1]);
+	if ($some_post instanceof ElggEntity) {
+		$my_page_entity = $some_post;
+		$page_entity_type = $my_page_entity->getSubtype();
+	}
+	
+	if (!$some_post instanceof ElggEntity)
+		$some_post = get_entity($segments[sizeof($segments) - 2]);
+
+	if ($some_post instanceof ElggEntity) {
+		$my_page_entity = $some_post;
+		$page_entity_type = $my_page_entity->getSubtype();
+	}
+
+}
+
 ?>
 
-<?php if ($my_page_entity instanceof ElggEntity) { ?>
+
+<?php if ($my_page_entity instanceof ElggEntity) { 
+$description = strip_tags(gc_explode_translation($my_page_entity->description, 'en')) . strip_tags(gc_explode_translation($my_page_entity->description, 'fr'));
```

Comment:
This is entirely added in solely for the crawler, it was part of the requirement to be implemented so that the search engine captures and indexes all the information